metadata/grid.xml

```diff
@@ -30,52 +30,52 @@
 		<option name="slot_bl" type="activator">
 			<_short>Bottom left</_short>
 			<_long>Positions the window in the bottom left corner of the screen with the specified activator.  The window takes 1/4 of the screen.</_long>
-			<default>&lt;alt&gt; &lt;ctrl&gt; KEY_KP1</default>
+			<default>&lt;super&gt; &lt;shift&gt; KEY_HOME | &lt;super&gt; KEY_KP1</default>
```

Comment:
It seems conflicting that there's a set of keys with two modifiers and a set with one, to do the same action. I would rather drop the former set in favor of the latter alone, because the key positioning doesn't make much sense on a traditional keyboard layout.

Reply:
The grid functionality is unreachable without a keypad.

Reply:
Here I agree with @alexherbo2, these make sense for a 14' laptop :)

However I think an even better solution would be: Retain the keypad bindings for big keyboards.
For small keyboards, add only left/right/center slots with <super> + ARROW, and make <super> + DOWN restore the window (the rest are only reachable via a keypad). These actions match the behavior of other DEs like GNOME.